opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -114,6 +114,25 @@ def shutdown(self) -> None:
         for sp in self._span_processors:
             sp.shutdown()
 
+    def force_flush(self, timeout_millis: int = None) -> bool:
+        flushed_within_time = True
+        flush_start_ns = time_ns()
+        for sp in self._span_processors:
+            if timeout_millis is None:
+                flushed_within_time = sp.force_flush() and flushed_within_time
```

Comment:
`timeout_millis` `None` should indicate that the force_flush could block forever, but `force_flush` in the span processor has a default timeout of 30 seconds. 

I think we should avoid providing an option for not having a timeout or update the span processors to implement it.

Reply:
my initial idea was to allow individual default timeouts for each span processor (e.g. someone implementing their own span processor) in `MultiSpanProcessor` but i guess that makes it just more confusing.
i'll go with adding a default (not `None`) timeout